src/System.Diagnostics.Process/tests/ProcessTests.netstandard1.7.cs

```diff
@@ -109,5 +110,74 @@ public void TestWorkingSet()
 
             Assert.True(_process.WorkingSet > 0);
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Process_StartInvalidNamesTest()
+        {
+            Assert.Throws<InvalidOperationException>(() => Process.Start(null, "userName", new SecureString(), "thisDomain"));
+            Assert.Throws<InvalidOperationException>(() => Process.Start(string.Empty, "userName", new SecureString(), "thisDomain"));
+            Assert.Throws<Win32Exception>(() => Process.Start("exe", "userName", new SecureString(), "thisDomain"));
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Process_StartWithInvalidUserNamePassword()
+        {
+            SecureString password = AsSecureString("Value");
+
+            Assert.Throws<Win32Exception>(() => Process.Start("exe", "userName", password, "thisDomain"));
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Process_StartTest()
+        {
+            string currentProcessName = $"{Process.GetCurrentProcess().ProcessName}.exe";
+            string userName = string.Empty;
```

Comment:
you should be able to pass the current user as the user.
seems you could potentially have two other tests: pass a garbage user (should fail) and pass current user with garbage password (should fail)